### PR TITLE
Issue 30999 fix

### PR DESCRIPTION
**Pull request by Dxin-code**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by alimcmaster1** [comment]
Please can you give all these descriptive titles + fix up pep8? Thanks

**Review by simonjayhawkins — CHANGES_REQUESTED** [review]
Thanks @Dxin-code for the PR. https://github.com/pandas-dev/pandas/pull/33581#issuecomment-614448854

File: pandas/tests/arrays/boolean/test_arithmetic.py [hunk 1]
```diff
@@ -23,20 +23,25 @@ def test_error(self, data, all_arithmetic_operators):
         opa = getattr(data, op)
 
         # invalid scalars
-        with pytest.raises(TypeError):
+         msg="invalid scalars"
```
Comment (jbrockmendel) [inline pandas/tests/arrays/boolean/test_arithmetic.py:26]: you've got a SyntaxError here

**Comment by simonjayhawkins** [comment]
@Dxin-code closing as stale.

**Pull request closed without merging**